openquake/mbt/tools/fault_modeler/fault_modeling_utils.py

```diff
@@ -2486,6 +2535,13 @@ def calc_mfd_from_fault_params(fault_dict,
     :type M_min:
         float
 
+    :param M_ref:
```

Comment:
I would replace the description with: 
In a  'DoubleTruncatedGR' we calculate the a-value given b-value, and two magnitude extremes: M_max and M_ref. Note that M_ref <= M_min.

Reply:
ok